ch2/Makefile

```
@@ -4,3 +4,6 @@ init:
 
 run-dependency:
```

```suggestion
dependency:
```
minor 한건데 굳이 앞에 `run` 을 붙이지 않아도 될 것 같습니다. `make run dependency` 로 봐도 동사가 두개 나와서 좀 이상한 것 같기도 하구요. 밑에 clean과도 비슷하게 가면 좋을 것 같아요.